test_suite/conftest.py

```diff
@@ -32,15 +39,36 @@ def check_markers(host, request):
     ]
 
     host_distro = host.system_info.distribution
-    host_version = host.system_info.release
+    host_version = float(host.system_info.release)
     distro_version = f'{host_distro}{host_version}'
 
     # Skip the test if the distro is explicitly excluded
     if exclude_on_marker:
         exclude_on_marker_list = exclude_on_marker.args[0]
         if host_distro in exclude_on_marker_list \
                 or distro_version in exclude_on_marker_list:
-            pytest.skip(f"This test doesn't apply to {distro_version}")
+            pytest.skip(skip_massage.format(distro_version=distro_version))
```

Comment:
I think there's a typo here. I would never skip a massage ;)

Reply:
The 'skip_message' is just a string, I noticed that it is repeated many times in the code so I think it is better to set one parameter for that.
Since this is the comment of the"pytest.skip", I thought that parameter named "skip_message" will define the operation - may another name be more appropriate?
skip_massage = "This test doesn't apply to {distro_version}"

Reply:
Sorry if I did not explain myself clear enough. My message was only about the fact that the variable has a typo and it is called "skip_massage" instead of "skip_message".

Reply:
got you:)
fixing it

Reply:
Thanks I just found the typo funny in this case because it's like skipping a massage session (which I would not skip because I love massages) LOL.